torch/_export/serde/serialize.py

```diff
@@ -519,6 +520,18 @@ def handle_call_function(self, node: torch.fx.Node):
                 outputs=self.serialize_hoo_outputs(node),
                 metadata=self.serialize_metadata(node),
             )
+        elif isinstance(node.target, CustomOpHandler):
+            custom_op_handler = node.target
+
+            # Sanity check for unhandled serialization.
+            assert type(node.target) in _serialization_registry, f"Miss {type(node.target)} custom op seralization"
+
+            ex_node = Node(
+                target=_serialization_registry[type(node.target)](node.target),  # Jump to custom serialization function.
```

Comment:
We can define a custom serialization format here to make deserialization parsing a bit easier:
```suggestion
            handler = _serialization_registry[type(node.target)]
            ex_node = Node(
                target=f"${handler.namespace()}:{handler.op_name(node.target)}",  # Jump to custom serialization function.
```